js/logic.js

```diff
@@ -1,13 +1,25 @@
+var map = require("./map.js");
+
 var logicObj = {
 
   resultsObj: {},
 
+  formatLineName: function(line){
```

Comment:
Does this reformat the name of the tube-line options to be lowercase? It is possibly easier to change the value attribute in the html form to what you actually want e.g. : `<option value="victoria">Victoria Line</option>`